field/src/field.rs

```diff
@@ -143,6 +144,12 @@ pub trait Field:
         }
         res
     }
+
+    // The frobenius operation to a field, for standard field, it returns itself.
+    // need to rewrite for complex field
+    fn base_frobenius(&self) -> Self {
+        *self
```

Comment:
I don't think we should provide a default impl, could be a bit dangerous as it'd be easy to forget to override it.

Reply:
I see. currently, I am thinking of dropping this thing due to the complex extension field is indeed not OEF, and its inverse, if using Frobenius, will have a different 'r' compare to the current implementation. The whole OEF trait is useful only when it is built upon prime field assumption. For other fields, should we have another more general trait and use algorithms  like 11.3.6 in the book <Handbook And Hyperelliptic Curve Cryptography>